docs/styling.md

```diff
@@ -6,7 +6,7 @@ DocSearch default colorscheme comes in a grey theme with blue highlight.
 
 ## Colorscheme
 
-<img src="https://docsearch.algolia.com/img/assets/default-colorscheme.png" alt="Default colorscheme"/>
+<img src="https://docsearch.algolia.com/img/assets/default-colorscheme.png" width="550px" alt="Default colorscheme"/>
```

Comment:
```suggestion
<img src="https://docsearch.algolia.com/img/assets/default-colorscheme.png" alt="Default colorscheme"/>
```
Let's avoid to have raw size within the code. I do not see any benefit of setting this size from a .md file.

Reply:
what do you mean ?

Reply:
`width="550px"`

Reply:
I'm curious about the "Let's avoid to have raw size".
For instance this image has a retina resolution, on a regular screen it's displayed too big and blurry. We should probably fix that with a better retina support but i'm curious about this practice of not having any width for images in .md

Reply:
I guess @s-pace's point was to either resize the source image or use CSS styling.

Reply:
oh I see, github markdown engine doesn't handle other attribute than alt. ok
makes sens now :)

Reply:
Exactly. Having a hardcoded width here will not solve the retina support. I do agree we would need to enhance this support. Another point is that one can view this .md from GitHub. We do not want to force the width.
![image](https://user-images.githubusercontent.com/32097720/69250853-bbbb8600-0bb0-11ea-8ea0-d3414f1ec910.png)

Reply:
Alright point solved after IRL discussion. The image was captured on a retina screen with a 2X resolution. Setting `width` attribute of an image is a good practice and this value is the half of the original image width.